lib/ex_force.ex

```diff
@@ -4,12 +4,14 @@ defmodule ExForce do
 
   ## Installation
 
-  The package can be installed by adding `ex_force` to your list of dependencies in `mix.exs`:
+  The package can be installed by adding `ex_force` and `tesla` (optional if you implement your own ExForce.Client) to your list of dependencies in `mix.exs`:
 
   ```elixir
   def deps do
     [
-      {:ex_force, "~> 0.3"}
+      {:ex_force, "~> 0.3"},
+      # Optional if you implement your own ExForce.Client
```

Comment:
The current code still *require* tesla. To make it optional, we need to load the default client only when tesla is available. Let's support "bring your own client module" later - let's keep requiring tesla for now.

Reply:
Fixed with 192868dd0d25388b6d23f6c77398f78e606413ce